ui/src/layout/Events/EventDetails/index.js

```diff
@@ -26,33 +28,42 @@ const EventDetails = () => {
         return null;
     }
 
-    const {time, hasProvidedSpecDiff, hasReconstructedSpecDiff} = data;
-    
+    const {time} = data;
+    const plugins = getPlugins(PLUGIN_TYPES.EVENT_DETAILS);
+    const pluginTabs = plugins.map((p) => {
```

Comment:
There is a very similar function in Inventory details, maybe move to a utils function